eZ/Publish/Core/REST/Client/ObjectStateService.php

```diff
@@ -251,15 +233,9 @@ public function createObjectState(ObjectStateGroup $objectStateGroup, ObjectStat
     }
 
     /**
-     * Loads an object state.
-     *
-     * @param mixed $stateId
-     *
-     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if the state was not found
-     *
-     * @return \eZ\Publish\API\Repository\Values\ObjectState\ObjectState
+     * {@inheritdoc}
      */
-    public function loadObjectState($stateId)
+    public function loadObjectState($stateId, array $prioritizedLanguages = [])
```

Comment:
@andrerom: I'm not sure how to pass prioritized languages to REST request. However you've mentioned that there are parts related to REST Client that we don't support (are broken), right? So maybe I shouldn't worry?